frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -1,54 +1,92 @@
-import { LemonDivider, LemonInput, LemonSelect } from '@posthog/lemon-ui'
+import { LemonButton, LemonDivider, LemonSelect } from '@posthog/lemon-ui'
 import { sdksLogic } from './sdksLogic'
-import { LemonCard } from 'lib/lemon-ui/LemonCard/LemonCard'
 import { useActions, useValues } from 'kea'
 import { OnboardingStep } from '../OnboardingStep'
 import { SDKSnippet } from './SDKSnippet'
 import { onboardingLogic } from '../onboardingLogic'
+import { useEffect } from 'react'
+import React from 'react'
+import { SDKInstructionsMap } from '~/types'
 
-export function SDKs({ usersAction }: { usersAction?: string }): JSX.Element {
-    const { setSourceFilter, setSelectedSDK } = useActions(sdksLogic)
+export function SDKs({
+    usersAction,
+    sdkInstructionMap,
+}: {
+    usersAction?: string
+    sdkInstructionMap: SDKInstructionsMap
+}): JSX.Element {
+    const { setSourceFilter, setSelectedSDK, setAvailableSDKInstructionsMap } = useActions(sdksLogic)
     const { sourceFilter, sdks, selectedSDK, sourceOptions } = useValues(sdksLogic)
     const { productKey } = useValues(onboardingLogic)
 
+    useEffect(() => {
+        setAvailableSDKInstructionsMap(sdkInstructionMap)
+    }, [])
+
     return (
         <OnboardingStep
             title={`Where are you ${usersAction || 'collecting data'} from?`}
-            subtitle="Pick one to start and add more sources later."
+            subtitle="Pick one or two to start and add more sources later."
         >
-            <div className="flex gap-x-4">
-                <LemonInput placeholder="Search for a source" type="search" />
-                {sourceOptions.length > 1 && (
-                    <LemonSelect
-                        allowClear
-                        onChange={(v) => setSourceFilter(v)}
-                        options={sourceOptions}
-                        placeholder="Select a source type"
-                        value={sourceFilter}
-                    />
+            <LemonDivider className="my-8" />
+            <div className="flex gap-x-8 mt-8">
+                <div className={`flex flex-col gap-y-2 flex-wrap gap-x-4 min-w-50`}>
+                    {sourceOptions.length > 1 && (
+                        <LemonSelect
```

Comment:
I believe this is a Lemon issue and nothing you have to worry about but the icon has a funky hover state<img width="225" alt="Screenshot 2023-09-06 at 11 14 55" src="https://github.com/PostHog/posthog/assets/6685876/87ef4e5f-4c96-4815-ace6-70457dab71a1">

Reply:
Actually that I play around with it more I'm not sure we need the select at all. Given we only have the one source (web) it's probably ok just to list all the frameworks we support

Reply:
Removed if there are few options 👍